test/libsolidity/util/BytesUtils.cpp

```diff
@@ -206,6 +225,19 @@ string BytesUtils::formatString(bytes const& _bytes, size_t _cutOff)
 	return os.str();
 }
 
+std::string BytesUtils::formatFixedPoint(bytes const& _bytes, size_t _fixedPointM, size_t _fixedPointN)
+{
+	solAssert(_fixedPointM != 0, "");
+	u256 max{boost::multiprecision::pow(u256{2}, static_cast<unsigned>(_fixedPointM))};
+	solAssert(u256{_bytes} >= max, "");
+	stringstream os;
+	os << u256{_bytes};
```

Comment:
What is `_bytes`? Is it binary or a decimal string representation?

You can use ``u256::str()`` and don't need the stringstream.